XLS-0070d-credentials/README.md

```diff
@@ -0,0 +1,573 @@
+
+A [verifiable credential](https://en.wikipedia.org/wiki/Verifiable_credentials) (VC), as defined by the [W3C specification](https://www.w3.org/TR/vc-data-model-2.0/), is a secure and tamper-evident way to represent information about a subject, such as an individual, organization, or even an IoT device. These credentials are issued by a trusted entity and can be verified by third parties without directly involving the issuer at all.
+
+For a clearer understanding of the relationship between [DIDs (Decentralized Identifiers)](https://en.wikipedia.org/wiki/Decentralized_identifier) and VCs, a real-world example is considered. A DID serves as a unique identifier for an entity, similar to a fingerprint or a photo of a person (their physical characteristics). In contrast, a VC acts as a verifiable piece of information associated with that DID, much like a driver's license or passport. When a third party needs to verify an identity claim, they can examine the VC presented by the entity and ensure it aligns with the corresponding DID (such as comparing the photo on a driver's license to the physical characteristics of the person to whom it supposedly belongs).
+
+### 1.2. Terminology
+
+* **Credential**: A representation of one or more assertions made by an issuer. For example, a passport is a representation of the assertion that a person is a citizen of a given country.
+* **Issuer**: The account that creates (issues) the credential. For example, a passport is issued by a country's government.
+* **Subject**: The account that the credential is issued to. For example, a passport is issued to a specific person.
+* **Whitelist**: A list of accounts that are allowed to do a given action, or (if used as a verb) the action of adding an account to said list.
+* **Preauthorization**: Essentially the same thing as whitelisting.
+
+*Note: These definitions do not directly match the definitions listed in the [W3C Verifiable Credentials spec](https://www.w3.org/TR/vc-data-model-2.0/#terminology). These terms are used in a slightly different way in the context of this spec.* 
+
+### 1.3. Basic Flow
+
+In this scenario, an authorizer/verifier account, Verity, is a regulated entity wants to ensure that she is only interacting with properly KYC'd accounts.
+
+* Verity sets up her account such that she only wants to interact with properly KYC'd accounts. He trusts Isabel, a credential issuer, to properly vet accounts and issue credentials attesting to that.
+* Alice, who wants to interact with Verity's account, submits her KYC documents to Isabel off-chain. Isabel creates a credential attesting to her trustworthiness on-chain.
+* Alice accepts the credential, making it a valid credential.
+* Alice can now interact with/send funds to Verity.
+
+## 2. On-Ledger Object: `Credential`
+
+This object will be an on-chain representation of a credential, while ensuring that any sensitive details are kept off-chain, to protect the subject's privacy.
```

Comment:
Future tense isn't necessary here. Also, I suggest adding a note about the owner reserve here.

```suggestion
This ledger entry is an on-chain representation of a credential. Sensitive details are kept off-chain, to protect the subject's privacy.

This entry counts as one item for purposes of the owner reserve towards either the issuer or the subject of the credential, depending on whether the subject has accepted it.
```